src/drivers/test/encoder_odometry_node.test

```diff
@@ -0,0 +1,19 @@
+<!-- ROSTests for the encoder_odometry_node -->
+
+<launch>
+
+    <!-- Start the Encoder to Odometry Node -->
+    <!-- TODO: THis should *not* output to "screen" -->
```

Comment:
Is this `TODO` still relevant?
Also noticed that `output="screen"` is also present in `encoder_odometry_node.launch` as well.

Reply:
Welp, glad you're catching all the TODO's I missed lol. The reason I didn't want this one to output to screen is that it's a test, and so we want to keep the test logs clean so that it's easy to see/find failures when running lots of tests (for example in CI). I don't think there's too much issue with having normal launch files outputting to `screen`. Changed: removed from `encoder_odometry_node.test` but **not** from `encoder_odometry_node.launch`